cmd/checkcommits/checkcommits_test.go

```diff
@@ -642,7 +642,7 @@ func TestGetCommitAndBranch(t *testing.T) {
 		}
 
 		// XXX: crucially, no arguments (to trigger the auto-detection)
-		commit, dstBranch, err := getCommitAndBranch([]string{}, []string{})
+		commit, dstBranch, _ := getCommitAndBranch([]string{}, []string{})
```

Comment:
ok, np, I'll rework. Yep, I did just pretty blindly apply the 'is ignored' ruling. Must remember to engage brain a little more next time